Client/src/App.jsx

```diff
@@ -1,5 +1,16 @@
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
```

Comment:
where did all the routes go?

Reply:
the routes for routing signup page to login and routing login page to signup page

Reply:
yeah there's none of them in this screen though